tox.ini

```diff
@@ -46,6 +46,7 @@ deps =
     oldestdeps: asdf-astropy==0.3.*
     oldestdeps: asdf==2.10.*
     oldestdeps: ndcube==2.0.*
+    oldestdeps: matplotlib==3.6.*
```

Comment:
If this is really the oldest supported version (and is it?), should that be reflected in the requirements in `setup.cfg`?
Asking since mamba wanted matplotlib 3.5.* for numpy 1.19, and tests are still passing with 3.5.3.

Reply:
No, that is not the oldest supported version. It is actually the maxversion we can allow for matplotlib for the oldest supported version. According to setup.cfg , matplotlib is only a test dependency.

Reply:
Yes, but if installed as test dependency (and not necessarily though tox), should not that also ensure a supported version is installed? And now that a version is specified in `oldestdeps`, I would expect, well, the actual oldest supported version to be tested (may be 3.1.0, which is the oldest I could reliably install and test with Python 3.8, or even older); just as it is done for numpy etc.

Reply:
Well, actually the tests are running (with the same 435 passing w/o remote_data) without any matplotlib installed, as it is only ever imported in the docs. So maybe the question rather is whether it belongs into `test` deps at all (especially with no doctests configured).

Reply:
Pretty sure specutils runs doctest because I just fixed those.

Reply:
I don't want to get into the debate of minversion of matplotlib specutils is supposed to support. I don't think that is very relevant. Would a comment on this line in `tox.ini` enough?

Reply:
If it is nowhere tested but just used for plotting out put in the docs, I suppose the version is not critical.
Still wondering if it is then needed as `test` dep at all, since `linkcheck` (which is the only doc-related test I could find) already pulls in `extras: docs` anyway.

Reply:
(there existed a `build_docs` job at the time when #849 was open, but I cannot see any obvious replacement for that)

Reply:
I think this setting pulls in `test` dependencies for all the jobs:

https://github.com/astropy/specutils/blob/6eb7f96498072882c97763d4cd10e07cf81b6d33/tox.ini#L62-L63

Reply:
Yes, that is clear; was just wondering if anything running doctests would also pull in `extras = docs` as `linkcheck` does. Answer is 'no'; turns out I misread the
`!cov: pytest --pyargs specutils {toxinidir}/docs {posargs}`
command for a commented line! (Maybe one should go Spanish and use `¡` for `not` ;-)

Reply:
So, do you still need a comment in `tox.ini`?

Reply:
I have reconstructed now how it is a documentation dependency that is required for all tests; no strong opinion either way if no one else has stumbled over it.